cmd/cluster-node-tuning-operator/main.go

```diff
@@ -25,24 +27,36 @@ func printVersion() {
 }
 
 func main() {
-	klog.InitFlags(nil)
-	flag.Parse()
+	operatorExec := regexp.MustCompile(`.*cluster-node-tuning-operator$`)
```

Comment:
Do we need to use regexp for this?  Please consider "path/filepath" Base().

Reply:
Good call